lib/pymedphys/docs/lib/howto/gamma/1D-from-csv.ipynb

```diff
@@ -0,0 +1,230 @@
+{
```

Comment:
<p>I really liked the improvements to the plot :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymedphys/pymedphys/pull/1603/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>